simple/msg_pingpong.c

```diff
@@ -40,6 +44,8 @@
 #include <rdma/fi_cm.h>
 #include <shared.h>
 
+static uint32_t FI_VER = FI_VERSION(1, 0);
```

Comment:
I wanted this up toward the top versioning  for 1.0, or even in shared.h so all the tests can reference it. You no like?